runtime/queries/glimmer/highlights.scm

```diff
@@ -0,0 +1,94 @@
+; === Tag Names ===
+
+; Tags that start with a lower case letter are HTML tags
+; We'll also use this highlighting for named blocks (which start with `:`)
+((tag_name) @tag
+  (#match? @tag "^(:)?[a-z]"))
+; Tags that start with a capital letter are Glimmer components
+((tag_name) @constructor
+  (#match? @constructor "^[A-Z]"))
+
+(attribute_name) @attribute
+
+(string_literal) @string
+(number_literal) @constant.numeric.integer
+(boolean_literal) @constant.builtin.boolean
+
+(concat_statement) @string
+
+; === Block Statements ===
+
+; Highlight the brackets
+(block_statement_start) @punctuation.delimiter
+(block_statement_end) @punctuation.delimiter
+
+; Highlight `if`/`each`/`let`
+(block_statement_start path: (identifier) @keyword.control.conditional)
+(block_statement_end path: (identifier) @keyword.control.conditional)
+((mustache_statement (identifier) @keyword.control.conditional)
+ (#match? @keyword.control.conditional "else"))
```

Comment:
These `#match?`es that don't use any regex syntax can be an `#eq?` instead. For example

```suggestion
 (#eq? @keyword.control.conditional "else"))
```

Reply:
implemented here: https://github.com/helix-editor/helix/pull/9902/commits/88ebcf010892a4dcdb429abcb0c95abadb2fa45f